Arrays/reshapemat.py

```diff
@@ -0,0 +1,36 @@
+def reshape(mat,a,b):
+    '''for i in range(a):
+        j=i
+        for j in range(b):
+            print(mat[j])'''
+    flat=[]
+    for ele in mat:
+        flat.extend(ele)
+     
+    for i in range(a*b):
+        if i%b==0and i!=0:
+            print("")
+        print(flat[i],end=" ")
+        
+        
+        
+m,n=input().split()
+a,b= input().split()
+mat=[]
+m=int(m)
+n=int(n)
+a=int(a)
+b=int(b)
+for i in range(m):
+    
+    list1=list(int(num) for num in input().strip().split())[:n]
```

Comment:
Do proper naming. List1 can be replaced with temp (standard for temporary list), makes for good coding practice.